octoprint_slic3r/__init__.py

```diff
@@ -274,10 +274,23 @@ def do_slice(self, model_path, printer_profile, machinecode_path=None, profile_p
 		if not executable:
 			return False, "Path to Slic3r is not configured "
 
+		args = ['"%s"' % executable, '--load', '"%s"' % profile_path, '--print-center', '"%f,%f"' % (posX, posY), '-o', '"%s"' % machinecode_path, '"%s"' % model_path]
+
+		try:
+			import subprocess
+
+			help_process = subprocess.Popen((executable, '--help'), stdout=subprocess.PIPE)
```

Comment:
Could you use `--version` instead of `--help`.  On my computer, with older Prusa, it says 1.39.2.

Reply:
On my build of 2.0.0 for the rpi the --version option appears to have been removed.